spec/services/discovery_engine/autocomplete/complete_spec.rb

```diff
@@ -0,0 +1,40 @@
+RSpec.describe DiscoveryEngine::Autocomplete::Complete do
+  subject(:completion) { described_class.new(query, client:) }
+
+  let(:client) { double("CompletionService::Client", complete_query:) }
```

Comment:
should this be an instance double?

Reply:
Good question – there is a factory method in the Google client library that gives you a client object back, without necessarily a guarantee that it's of a specific class (we do the same in the tests for `DiscoveryEngine::Query::Search`).

I've renamed the double name to be less class-looking though 👍